src/fastoad/io/configuration/configuration.py

```diff
@@ -115,6 +113,10 @@ def get_problem(self, read_inputs: bool = False, auto_scaling: bool = False) ->
 
         problem = FASTOADProblem()
         self._build_model(problem)
```

Comment:
It is not a change in the PR but I'm wondering here if this line means that we rebuild the problem each time we call the `get_problem` method. For instance if we were to keep calling the `write_needed_inputs` from the `FASTOADProblemConfigurator` class we would construct the problem twice.

Reply:
Indeed, the problem is built each time `get_problem` is called. I think it's better like this because:
- if a user called `get_problem`, he is able to keep the generated instance, and I think he has no reason to call `get_problem` again unless he wants a fresh new instance.
- we could imagine a case where the configuration is read, `get_problem` is called, then the configuration is programmatically modified (e.g. optimization definition) and `get_problem` is called again. In such case, a fresh new instance is really better.

This is why `write_needed_inputs` implementation has been moved to the `FASTOADProblem` class, which allows doing this operation on the instance that will be used for the computation.